cmd/trcli/activate/activate.go

```diff
@@ -42,6 +42,11 @@ func activate(
 			return xerrors.Errorf("unable to load transfer: %w", err)
 		}
 		transfer.Runtime = rt
+
+		if transfer.Telemetry != nil && transfer.Telemetry.Prefix != "" {
+			registry = registry.WithPrefix(transfer.Telemetry.Prefix)
+		}
+
 		return RunActivate(*cp, transfer, registry, delay)
```

Comment:
see this delay option, you can replace this var with something like `TelemetryOpts` and pass this prefix there.